code/obj/item/organ_holder.dm

```diff
@@ -826,6 +826,14 @@
 						H.wear_mask = newHead.wear_mask
 						newHead.wear_mask.set_loc(H)
 						newHead.wear_mask = null
+					if (isskeleton(H))
+						var/datum/mutantrace/skeleton/S = H.mutantrace
+						if (S.head_tracker != null)
+							S.head_tracker.linked_human = null
+						newHead.head_type = HEAD_SKELETON
+						newHead.linked_human = H
```

Comment:
`set_head` covers this.

Reply:
I'm not sure it does?
It does set `src.head_tracker = head` but not linked_human i believe?

Reply:
Can you move the setting of `linked_human` to under `set_head`?

Reply:
Done so!